execute_process_mass_data.py

```diff
@@ -98,7 +98,7 @@
 # ----- Choose what to save ------------------------
 # --------------------------------------------------
 element_pmf_results_df = pd.DataFrame(data=element_pmf_results)
-save_element_pmf_data(element_pmf_results_df, stdev, mrange)
+# save_element_pmf_data(element_pmf_results_df, stdev, mrange)
```

Comment:
commenting out / uncommenting lines like this is a sign that we need a config file or a command or something?